cli/lsp/analysis.rs

```diff
@@ -60,7 +60,7 @@ static PREFERRED_FIXES: Lazy<HashMap<&'static str, (u32, bool)>> =
   });
 
 static IMPORT_SPECIFIER_RE: Lazy<Regex> =
-  Lazy::new(|| Regex::new(r#"\sfrom\s+["']([^"']*)["']"#).unwrap());
+  lazy_regex::lazy_regex!(r#"\sfrom\s+["']([^"']*)["']"#);
```

Comment:
What's the advantage of using `lazy_regex` over current solution?

Reply:
I think we should keep using once_cell only because it's an easy solution and doesn't involve adding another crate.

Reply:
I tend to agree

Reply:
`lazy_regex` just provide some macro which is widely used(65k+ downloaded) and small.

The macro kind solution is also recommended by [once_cell](https://docs.rs/once_cell/latest/once_cell/#lazily-compiled-regex).

Reply:
About the advantages, it

+ provide same features with less code.
+ no unwrap.
+ At most cases, we should create a cached regex, other code contributors can follow the specification.

Reply:
It does look nice, but it’s yet another crate that could go stale or get out of sync with our shared dependencies and using a lazy is not a big deal.

We should remove the usages of lazy_static and cache that one regex though.

Reply:
I agree; it doesnt seem necessary for no real benefit

Reply:
ok, I will fix it